solr/modules/sql/src/java/org/apache/solr/handler/sql/SolrFilter.java

```diff
@@ -362,8 +362,9 @@ protected String translateLike(RexNode like) {
         // but that expects the terms wrapped in double-quotes, not parens
         boolean hasMultipleTerms = terms.split("\\s+").length > 1;
         if (hasMultipleTerms && (terms.contains("*") || terms.contains("?"))) {
-          String quotedTerms = "\"" + terms.substring(1, terms.length() - 1) + "\"";
-          return "{!complexphrase}" + pair.getKey() + ":" + quotedTerms;
+          String quotedTerms = "\\\"" + terms.substring(1, terms.length() - 1) + "\\\"";
+          String query = pair.getKey() + ":" + quotedTerms;
+          return String.format("{!complexphrase v=\"%s\"}", query);
```

Comment:
```suggestion
          return String.format(Locale.ROOT, "{!complexphrase v=\"%s\"}", query);
```
This should fix the precommit failure